backend/tests/endpoints/test_message.py

```diff
@@ -2,3 +2,45 @@
 from main import app
 
 client = TestClient(app)
+
+fake_room_id = "44523d5325dsdf"
+fake_org_id = "1223209805sdfsa902"
+fake_message_data = {
+        "sender_id": "425324132312",
+        "room_id": fake_room_id,
+        "text": "does it work well?",
+        "reactions": [],
+        "files": [],
+        "saved_by": [],
+        "is_pinned": False,
+        "is_edited": False,
+        "created_at": "2021-12-11 01:44:37.582879",
+        "threads": []
+        }
+
+def test_send_message():
+    """ test for the send message endpoint"""
+
+    fake_response_output = {
+        "message_id": "33090940900",
+        "message_data": fake_message_data
+    }
+
+    response = client.post(
```

Comment:
Nice work.
The only issue is that the test_send_message would actually be populating the db with test data.
This can be avoided if the actually write request was a mock function.
Or 
A test data can be sent. 
On validating that it was actually sent, a delete request is sent to the delete the message. so that the db is not populated. The issue with this is that your time taken for the tests to be carried would be longer.

Therefore, the best approach would be to mock the db.write function.

Reply:
Alright. 
Will work on the mock function later tonight.